buildlib/tools/coverity.sh

```diff
@@ -79,5 +92,14 @@ run_coverity() {
 	return $rc
 }
 
-prepare_build
+az_init_modules
```

Comment:
how does it work when running outside the CI? seems this script is not really standalone

Reply:
tbh, i did not see anything very az specific. there is az_module_load, but it's only logging that is AZ specific.

Reply:
can we remove it then?

Reply:
it's difficult because of all naming az_module_load which in practice does a module load with hypothetical `az_log_warning` on failure.

But after checking, even that logging function does finally check for AZ presence:
```
# Report an warning message to Azure pipeline
function azure_log_warning() {
    test "x$RUNNING_IN_AZURE" = "xno" && return
    msg=$1
    set +x
    echo "##vso[task.logissue type=warning]${msg}"
}
```

Reply:
tbh, even being able to trigger azure logs locally (setting RUNNING_IN_AZURE=1) could be useful when developping script

Reply:
ok